src/lib.rs

```diff
@@ -888,8 +885,8 @@ where
 impl Buffer {
     pub fn map_read_async<T, F>(&self, start: BufferAddress, size: BufferAddress, callback: F)
     where
-        T: 'static + FromBytes,
```

Comment:
why is `static` removed from here? We don't know when the call back is going to be called, and for this time frame `T` has to be valid. `static` is a conservative bound to enforce that